Luke7787/ImmaculateInventors
# Feature/imageitems

Summary:
When creating an item, we can now include an image

How to test:
npm run dev

In boomerang:
![image](https://github.com/Luke7787/ImmaculateInventors/assets/121963264/7e15a1a7-8548-439e-9247-c97c0f237367)

-dont change the names imageFile and updates
-the top field is for images
-the bottom field is for items
-choose an image from your computer
-change the values of the items, you can keep the folder part unchanged since there is already a folder in the db, or you can make a new one to test it

-check that the item got added into the correct user associated with the folder
-check that the item got added to the folder you specified
-check that the item got added and under the image field is the s3 url

Notes:
-if you are frontend reviewing this, the field for the item 
![image](https://github.com/Luke7787/ImmaculateInventors/assets/121963264/58fbd36f-4413-433d-a123-d24e7f909b42)
is a string that is converted into json later in the backend
